docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,271 @@
+      and [Amazon AWS S3](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html).
+
+      - Add Alluxio worker hostnames into `${ALLUXIO_HOME}/conf/workers`.
+
+         ``` json
+         NM_hostname_1
+         NM_hostname_2
+         ```
+
+      - Copy configuration from Alluxio master to Alluxio workers.
+
+         ``` shell
+         ${ALLUXIO_HOME}/bin/alluxio copyDir ${ALLUXIO_HOME}/conf
+         ```
+
+         This command will copy the `conf/` directory to all the workers specified in the `conf/workers` file.
+         Once this command succeeds, all the Alluxio nodes will be correctly configured.
+
+   - Alluxio worker configuration.
+
+      After copying configuration to every Alluxio worker from Alluxio master, User
+      needs to add below extra configuration for each Alluxio worker.
+
+      ``` xml
+      # the hostname of Alluxio worker
+      alluxio.worker.hostname=NM_hostname_X
+      # The hostname to use for an Alluxio client
+      alluxio.user.hostname=NM_hostname_X
+      ```
+
+      Note that Alluxio can manage other storage media (e.g. MEM, HDD) in addition to SSD,
+      so local data access speed may vary depending on the local storage media. To learn
+      more about this topic, please refer to the
+      [tiered storage document](https://docs.alluxio.io/os/user/stable/en/core-services/Caching.html#multiple-tier-storage).
+
+5. Mount an existing S3 bucket to Alluxio.
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio fs mount \
+      --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> \
+      --option aws.secretKey=<AWS_SECRET_KEY_ID> \
+      alluxio://RM_hostname:19998/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
+   ```
+
+   For other filesystems, please refer to [this site](https://www.alluxio.io/).
+
+6. Start Alluxio cluster.
+
+   Login to Alluxio master node, and run
+
+   ``` bash
+   ${ALLUXIO_HOME}/bin/alluxio-start.sh all
+   ```
+
+   To verify that Alluxio is running, visit [http://RM_hostname:19999](http://RM_hostname:19999)
+   to see the status page of the Alluxio master.
+
+## RAPIDS Configuration
+
+There are two ways to leverage Alluxio in RAPIDS.
+
+1. Explicitly specify the Alluxio path.
+
+   This may require user to change code. For example, Change
+
+   ``` scala
+   val df = spark.read.parquet("s3a://<S3_BUCKET>/<S3_DIRECTORY>/foo.parquet")
+   ```
+
+   to
+
+   ``` scala
+   val df = spark.read.parquet("alluxio://RM_hostname:19998/s3/foo.parquet")
+   ```
+
+2. Transparently replace in RAPIDS.
+
+   RAPIDS has added a configuration `spark.rapids.alluxio.pathsToReplace` which can allow RAPIDS
+   to replace the input file paths to the Alluxio paths transparently at runtime. So there is no
+   code change for users.
+
+   Eg, at startup
+
+   ``` shell
+   --conf spark.rapids.alluxio.pathsToReplace="s3:/foo->alluxio://RM_hostname:19998/foo,gs:/bar->alluxio://RM_hostname:19998/bar"
+   ```
+
+   This configuration allows RAPIDS to replace any file paths prefixed`s3:/foo` to
```

Comment:
add space between prefixed and 's3.  Also change "to" to "with" and same with below "to" to "with"